bentoml/_internal/io_descriptors/pandas.py

```diff
@@ -16,10 +16,46 @@
 else:
     pd = LazyLoader("pd", globals(), "pandas")
 
+try:
+    from typing import Literal
+except ImportError:  # pragma: no cover
+    from typing_extensions import Literal
+
 logger = logging.getLogger(__name__)
 
 
-class PandasDataFrame(IODescriptor):
+def _infer_type(item: str) -> str:  # pragma: no cover
```

Comment:
seems like there are multiple infer type methods?

Reply:
only for pandas and numpy

Reply:
should those be unified or do they need to be distinct?

Reply:
distinct